src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -2209,7 +2203,11 @@ public Long zcount(final byte[] key, final byte[] min, final byte[] max) {
   private Set<Tuple> getBinaryTupledSet() {
     checkIsInMulti();
     List<byte[]> membersWithScores = client.getBinaryMultiBulkReply();
-    Set<Tuple> set = new LinkedHashSet<Tuple>();
+    if (membersWithScores.size() == 0) {
+      return Collections.emptySet();
+    }
+    Set<Tuple> set = new LinkedHashSet<Tuple>(
+        CollectionUtils.mapCapacity(membersWithScores.size() / 2));
```

Comment:
@marcosnils I do that to avoid resizing because some JDKs do recalculate by loadFactor.
http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/7-b147/java/util/HashMap.java?av=f#203

But because Oracle JDK use power of two size, my intention was wrong.
Just do `new LinkedHashSet<Tuple>(membersWithScores.size() / 2, 1.0)` is enough.
` / 2` is neccessary because a Tuple consume two elements of `membersWithScores`.